st2common/bin/st2-self-check

```diff
@@ -136,6 +142,11 @@ do
         continue
     fi
 
+    if [[ -n "${GITHUB_ACTIONS}" && " ${TESTS_TO_SKIP} " =~ " ${TEST} " ]]; then
```

Comment:
I just realized another glitch here. `-n "${GITHUB_ACTIONS}"` is no longer needed when we want to have a generic `TESTS_TO_SKIP`. I'll fix this.

Reply:
Done.